profiles/Mustafiz04.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/49086821/148987425-43f568f6-1a74-4720-a80e-45e025c1751d.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/148999399-fdaf15a3-607c-4a04-98fa-4fb1cda7a6df.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/148999399-fdaf15a3-607c-4a04-98fa-4fb1cda7a6df.png",
```

Reply:
I have make the changes.